OsmAnd-java/src/main/java/net/osmand/binary/RouteDataObject.java

```diff
@@ -1010,6 +1010,20 @@ private double simplifyDistance(int x, int y, int px, int py) {
 		return Math.abs(px - x) * 0.011d + Math.abs(py - y) * 0.01863d;
 	}
 
+	// 0..360 degrees
+	public double directionRouteByNeighbours(int index) {
```

Comment:
Should be used directionRoute(index, true, 5); // to avoid different weird situation already handled in directionRoute

Reply:
ok